isis/src/dev/apps/camcoeffs/camcoeffs.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="camcoeffs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd"> 
+  <brief>
+    This program calculates TRANSX,TRANSY from TRANSS, TRANSL for iak kernels
+  </brief>
+
+  <description>
+      This application is for camera development is not supported for general public use.

```

Comment:
Also, this line looks like it might be missing an _and_:
> This application is for camera development **and** is not supported for general public use.